queries/lua/highlights.scm

```diff
@@ -129,7 +129,13 @@
 (identifier) @variable
 
 ((identifier) @variable.builtin
- (#any-of? @variable.builtin "self" "string" "table" "vim"))
+ (#any-of? @variable.builtin "_G" "_VERSION" "io" "math" "os" "package" "self" "string" "table" "utf8" "vim"))
```

Comment:
We deliberately keep this parser and query generic (not specifically targeting for Neovim dev, unlike TJ's parser). So this should only be variables actually built in to the language.

Reply:
These are generic, see the index here - https://www.lua.org/manual/5.3/

The only exception is "vim", which is fine imo :)

Reply:
Yes, I was specifically talking about `vim`, of course. The rest is fine (and `vim` is not, in my view).

Reply:
(remember that we are talking about making it easier to share queries -- and this would be a move in the wrong direction).

Also, it's very easy to add that to an `; extends` query locally.

Reply:
Makes sense...that conversation has been a bit stale lately though :confused: seems like we need Helix to implement logic to use last wins, then we can standardize query captures/predicates...well that's getting off topic. I'll remove vim now